### ISSUE TITLE
SwiftyJSON not building, no error, no Xcode build output

### ISSUE THREAD

**Issue by petereddy** [Carthage/Carthage#532]
This is in version 0.7.4. I created a Cartfile with the following contents:

github "SwiftyJSON/SwiftyJSON"

$ carthage update
*** Fetching SwiftyJSON
*** Checking out SwiftyJSON at "2.2.0"
*** xcodebuild output can be found in /var/folders/qm/6l_xyqjs35b5zbhhnf4fyvy00000gn/T/carthage-xcodebuild.hiSkw1.log
$ cat /var/folders/qm/6l_xyqjs35b5zbhhnf4fyvy00000gn/T/carthage-xcodebuild.hiSkw1.log
$ 

In other words, no errors and the log is empty. If I add other projects to the Cartfile it builds them normally.

**Comment by jspahrsummers** [comment]
This is because Carthage is finding the `Example.xcodeproj` of SwiftyJSON, instead of the project file containing the framework itself. See https://github.com/Carthage/Carthage/issues/476 for a related discussion.

In the meantime, possible workarounds would be to rename `Example.xcodeproj` so it's sorted after the main project, or move it into a folder so it’s more nested.

**Comment by Thomvis** [comment]
I've created a PR for SwiftyJSON that 'fixes' this issue by moving the example project to a subdirectory: https://github.com/SwiftyJSON/SwiftyJSON/pull/244. I'll be using my own fork until it is merged: https://github.com/Thomvis/SwiftyJSON. I tagged it as 2.2.1.

**Comment by petereddy** [comment]
Thanks, Thomvis.

By the way, I created this issue because I thought it was bad to fail silently, not that I thought that carthage should be able to handle any odd project structure. 

It'd be nice if there were some message to the effect that the project's structure wasn't supported or something. The current behavior make it look like a bug in carthage.

**Comment by jspahrsummers** [comment]
See https://github.com/Carthage/Carthage/issues/476 and https://github.com/Carthage/Carthage/issues/108, which kinda cover this.

**Comment by hartbit** [comment]
> See #476 and #108, which kinda cover this.

I'd like to get this issue re-opened. I agree with @petereddy that it seems wrong to skip a dependency silently - so entirely a reporting issue for me. The issues you linked do provide more information, but nothing concerning this silent reporting.

**Comment by ikesyo** [comment]
@hartbit I'm working on it (#476) now. :full_moon_with_face:

**Comment by kortina** [comment]
agree w @hartbit

**Comment by pcantrell** [comment]
+1 for a more useful console message

**Comment by morizotter** [comment]
+1

**Comment by tualatrix** [comment]
Agree with more output or error from console message, at least: can't find framework scheme.

**Comment by ikesyo** [comment]
[0.8](https://github.com/Carthage/Carthage/releases/tag/0.8) is released, it should fix the issue.
